pkg/cloudprovider/vsphere/loadbalancer/README.md

```diff
@@ -0,0 +1,111 @@
+# NSX-T Load Balancer Controller
+
+*Kubernetes load balancer support using NSX-T for the `vSphere` cloud controller
+manager.*
+
+This package enriches the cloud provider interface by implementing the load
+balancing API of the cloud controller for an NSX-T environment.
+
+There are two different modes:
```

Comment:
Can we imply the different modes based on whether a loadbalancer service ID is provided or not in the config?

Reply:
Ah I see from the reading below this is the case right?

Reply:
For the first iteration I would actually prefer to always default to the "unmanaged mode" -- in other words already require a service ID from a user. As a future feature we can create the LB service automatically.

Reply:
The managed mode is important for us, because we want to do as less as possible for the automated infrastructure setup on our own.

Basically the mode must not explicitly be selected. So, if the service is specified, it is used. Therefore the standard usecase could be to specify the service and it works as expected.

We could mention the two modes less prominent in the docu and all examples just specify the service.

Would this be ok for you?

Reply:
> We could mention the two modes less prominent in the docu and all examples just specify the service.

Sounds good to me.

Reply:
done